projects/plugins/jetpack/tests/php/test_class.functions.opengraph.php

```diff
@@ -222,4 +222,64 @@ public function test_jetpack_og_get_image_from_post_order() {
 		$first_image_url = $post_info['img_urls'][0];
 		$this->assertEquals( $first_image_url, $chosen_image['src'] );
 	}
+
+	/**
+	 * Helper function to get default alt text.
+	 *
+	 * @return string
+	 */
+	public function get_default_alt_text() {
+		return 'Default alt text';
+	}
+
+	/**
+	 * Test if jetpack_og_get_image returns the correct default alt text.
+	 *
+	 * @author automattic
+	 * @covers ::jetpack_og_get_image
+	 * @since  $$next-version$$
```

Comment:
@anomiex are we able to use $$next-version$$ in tests, or is the check failing for a reason I can't grok?

Reply:
It doesn't like the extra space in there.
```suggestion
	 * @since $$next-version$$
```
Or we could adjust the tool if we want to allow for it.

Reply:
I thought phpcs might yell at us without having everything lined up, but it doesn't seem to care. Probably not a bad idea to adjust the tool, but for this PR we can just remove the spaces.

@danielpost You want to remove the spaces, merge with master again, and then we can merge this one?

Reply:
Sure thing!